src/api/auth.rs

```diff
@@ -88,7 +89,8 @@ pub async fn code(conn: DbConn, code: &str, cookies: &CookieJar<'_>) -> Result<R
                     // Create the user's personal team
                     let team = diesel::insert_into(teams)
                         .values(&NewTeam {
-                            name: format!("{}'s team", info.name),
+                            name: Some(format!("{}'s team", info.name)),
+                            slug: Generator::default().next().unwrap(),
```

Comment:
Can you make the slug the username instead as it makes it easier later on (as only to the user we have the global namespace, not in the DB internally)